src/components/muiCard.js

```diff
@@ -8,26 +8,21 @@ import {
   CardContent,
   CardMedia,
   Typography,
-} from '@mui/material';
-import {
-  Favorite,
-  FavoriteBorder,
-  BrokenHeart
-} from "@mui/icons-material";
+} from "@mui/material";
+import { Favorite, FavoriteBorder, BrokenHeart } from "@mui/icons-material";
 
 export default function MultiActionAreaCard({
   mapImage,
   mapTitle,
   mapDescription,
-  mapLink
+  mapLink,
 }) {
-
   const [starred, setStarred] = useState(false);
 
   return (
     <Card
       sx={{
-        bgcolor: "primary.main",
+        bgcolor: "secondary.main",
```

Comment:
cause you don't need to import theme here. I think you're just importing the JSON object and using its property. Maybe make a "style constants" file of some kind and keep the gradients there?